app/main.py

```diff
@@ -1,25 +1,18 @@
-from os import getenv
-
-from fastapi import FastAPI, Security
+from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
-from fastapi.responses import Response, JSONResponse
-from fastapi.requests import Request
 import uvicorn
+from app.api import wordcloud_text, wordcloud_cloud, wordcloud_update
 
-from app.api import submission, visualization, clustering, db
-from app.utils.security.header_checking import get_api_key
-
-app = FastAPI(
+application = app = FastAPI(
     title="Labs26-StorySquad-DS-Team B",
     description="A RESTful API for the Story Squad Project",
     version="0.1",
     docs_url="/"
 )
 
-app.include_router(submission.router, tags=['Submission'], dependencies=[Security(get_api_key)])
-app.include_router(visualization.router, tags=['Visualization'], dependencies=[Security(get_api_key)])
-app.include_router(clustering.router, tags=['Clustering'], dependencies=[Security(get_api_key)])
-app.include_router(db.router, tags=['Database'], dependencies=[Security(get_api_key)])
```

Comment:
We probably don't want to turn these off in the main branch of the repo.

Reply:
Good call, I put them back in.